src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -0,0 +1,285 @@
+import org.broadinstitute.hellbender.utils.io.Resource;
+import org.broadinstitute.hellbender.utils.python.PythonScriptExecutor;
+
+import java.io.File;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Model the baseline ploidy per contig for germline samples given their fragment counts.
+ * These should be either HDF5 or TSV count files generated by {@link CollectFragmentCounts}.
+ *
+ * <p>This tool has two run-modes as described below:</p>
+ * <dl>
+ *     <dt>COHORT run-mode</dt>
+ *     <dd>If a model parameter directory is not provided, the tool is run in the COHORT mode. In this run-mode,
+ *      ploidy model parameters (e.g. coverage bias and variance in each contig) are inferred
+ *      along with baseline contig ploidy of each sample. A table specifying priors for the ploidy per contig
+ *      is required in this run-mode.
+ *
+ *      The output will contain two subdirectories, one ending with "-model" and the other with "-calls".
+ *      The model subdirectory contains the inferred parameters of the ploidy model, which may be used later
+ *      for ploidy determination in one or more similarly-sequenced samples (see below).
+ *
+ *      The calls subdirectory contains one subdirectory for each sample, listing various sample-specific
+ *      quantities such as the global read-depth, average ploidy, per-contig baseline ploidies, and per-contig
+ *      coverage variance estimates.</dd>
+ *
+ *     <dt>CASE run-mode</dt>
+ *     <dd>If a previously obtained model parameter bundle is provided (e.g. from a previous run on a similar set
+ *      of samples), then the tool is run in the CASE mode. In this run-mode, the ploidy model parameters are
+ *      set to the given parameter bundle and only sample-specific quantities are estimated. Subsequently, the
+ *      output directory will contain only the "-calls" subdirectory and is structured as described above.
+ *
+ *      In this run-mode, the contig ploidy prior table is taken directly from the provided model parameters
+ *      path and must be not provided again.</dd>
+ * </dl>
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>COHORT run-mode:</p>
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" DetermineGermlineContigPloidy \
+ *   --input normal_1.counts.hdf5 \
+ *   --input normal_2.counts.hdf5 \
+ *   ... \
+ *   --output output_dir \
+ *   --output-prefix normal_cohort
+ * </pre>
+ *
+ * <p>CASE run-mode:</p>
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" DetermineGermlineContigPloidy \
+ *   --model a_valid_ploidy_model_dir
+ *   --input normal_1.counts.hdf5 \
+ *   --input normal_2.counts.hdf5 \
+ *   ... \
+ *   --output output_dir \
+ *   --output-prefix normal_case
+ * </pre>
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        summary = "Model the baseline ploidy per contig for germline samples given their fragment counts.",
```

Comment:
`fragment counts` -> just `counts` here and below.

Reply:
Done.